command/crypto/otp/verify.go

```diff
@@ -67,23 +73,66 @@ as "300ms", "-1.5h" or "2h45m". Valid time units are "ns", "us" (or "µs"), "ms"
 }
 
 func verifyAction(ctx *cli.Context) error {
-	args := ctx.Args()
-	secretFile := args.Get(0)
+	var (
+		secret string
+		// defaults
+		period = uint(30)
+		digits = 6
+		algStr = "SHA1"
+	)
 
+	secretFile := ctx.String("secret")
 	b, err := ioutil.ReadFile(secretFile)
 	if err != nil {
 		return errs.FileError(err, secretFile)
 	}
-	secret := string(b)
+	secret = string(b)
 	if strings.HasPrefix(secret, "otpauth://") {
```

Comment:
Just a question here. If the secret is passed as an url does it make sense to read it from a file? Or would something like this be more common?
```sh
step crypto otp verify otpauth://foo.bar.zar 
```
If it makes sense this usage, we can support both.

Reply:
I think this is more a matter of taste. From my point of view the "secret" is a password, so to be consistent with our other commands I think it makes sense to read it from a file. Open to your thoughts as well though.

Reply:
And if we require a file the the secret will never show up in your shell logs.

Reply:
Ok makes sense.